uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-assignment.js

```diff
@@ -49,6 +52,9 @@ define([
             $scope.activitySet = {};
             $scope.taskList = [];
             $scope.documentList = [];
+            $scope.activity = {
```

Comment:
I am not sure I understand why it should be nested, despite of the fact that you have mentioned it in the PR description. Could you please provide an explanation or a reference link that recommends to use nested objects for the scope.